pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -158,7 +158,6 @@ def _run_early_stopping_check(self, trainer, pl_module):
         if self.monitor_op(current - self.min_delta, self.best_score):
             self.best_score = current
             self.wait_count = 0
-            should_stop = False
         else:
             self.wait_count += 1
             should_stop = self.wait_count >= self.patience
```

Comment:
shall this be removed?

Reply:
`should_stop` is actually used two lines below:
https://github.com/PyTorchLightning/pytorch-lightning/blob/b47e411eadf39b8fb1104141779cd75ab1a63101/pytorch_lightning/callbacks/early_stopping.py#L165

Reply:
yes, my point is rather the mixing `should_stop` and `trainer. should_stop` which makes confusion

Reply:
Yes, the default should be whatever the trainer has already set because other early stopping callbacks may have already changed this value.

Reply:
@Borda i see what you mean, let me change the name of the value to clarify